### ISSUE TITLE
AAG talk

### ISSUE THREAD

**Issue by kzaret** [kzaret/RQ1v2_PIUVestab#7]
I'll be presenting during the Dendro I session that runs from 8 am to 9:15 am [uufff] on 4/9.

To refresh our memories, here's the title, abstract and keywords:

Dendroecology in ecotonal wetscapes:  Tree establishment dynamics in the forest-peatland ecotone of southwest Patagonia

Climate-fire synergies have the potential to shift forest ecosystems to non-forested states by reducing fire-free intervals while altering the environmental conditions that had been conducive to tree recruitment and survival. Though climate-fire-vegetation feedbacks have been investigated in many parts of the globe, few studies have focused on ecosystems that are both climatically and edaphically wet.  Here, we introduce our dendroecological research program at the forest-peatland ecotone of southwest Patagonia.  Using fire scar data and tree-ring counts from the long-lived conifer Pilgerodendron uviferum, we intend to compare post-fire tree establishment during climatically distinct periods pre- and post- Euro-Chilean settlement (i.e., variable cool/wet vs. warm/dry conditions from 1600 to 1920, cool/wet conditions from 1920 to 1950, and warm/dry conditions after 1950).  However, our field sites are characterized by deep organic soils or peat moss lawns that obscure tree bases, and P. uviferum often forms adventitious roots or a lateral growth habit, thus generating high levels of uncertainty in coring heights, ring counts and establishment dates.  We are currently exploring models (e.g., Bayesian state-space models that incorporate both process variation and observation error, including ageing uncertainty) that will allow us to increase the temporal resolution at which we examine tree establishment density as a function of time-since-fire and large-scale climate modes (e.g., Southern Annular Mode). This research will shed light on whether anthropogenically altered fire regimes, climate or their interactive effects are driving post-fire transitions from wet forests to non-forested peat-generating wetlands in southwest Patagonia and perhaps other oceanic temperate forest biomes.

Keywords:  forest-peatland ecotone, post-fire tree establishment, Patagonia, dendroecology, ecosystem transition

**Comment by ebuhle** [comment]
D'oh, not sure how AAG became AGU in my mind.

I'll be interested to see an outline of what you're thinking. I was going to suggest walking through the components of the age-correction modeling (coring-height correction, Duncan rings-to-pith, missing outer rings [TBD]) up to the preliminary results for priors on total tree age and recruitment density, and then concluding with a sneak preview of the time series modeling using my original toy simulations.

I'm relieved that we gave ourselves enough weasel words that all we have to do is "introduce our research program", so I'm not too worried about things being preliminary. However, that last transition might be tricky b/c the toy version of the time-uncertain Poisson state-space model differs from our intended use case in a subtle but important way: it takes "uncorrected time observations" as input and uses an observation likelihood, whereas we'll be putting an informative prior on time that incorporates the corrections. This could be confusing and is way too subtle to explain in a talk, but as long as you can keep that part vague and hand-wavy, it's probably fine. (Shifting between "time" and "age" might also be confusing, even though they're basically the same thing.)

**Comment by kzaret** [comment]
Well, one of my overarching goals of the talk is indeed to introduce my dissertation work to a large interested audience.  I presented a poster at the Southern Connection Congress in 2015 but other than that, nada.  So, I want to give a conceptual overview of the overarching goals of my dissertation, mention my various research questions and methods (dendro-, paleo-, neoecological), then focus on the specific research question driving the analysis of _P. uviferum_ establishment that we're working on right now.

I sketched the following for my last meeting with Andres based on what you'd written about how the components of the analysis come together.  Perhaps a version of this could be used in the talk.

![AnalysisApproachPIUVestab](https://user-images.githubusercontent.com/73861013/112375061-0ca92600-8ca0-11eb-9764-47e1d1ba1568.jpg)

In general, keeping things as simple as possible would be my preference since I've already entered "stage fright" mode and will only become increasingly anxious and less able to think clearly as April 9th approaches.  [I hear Bob Kaplan saying, "You have to play the game", and my response then as it is now is, "Actually, I don't."]

**Comment by ebuhle** [comment]
That all makes sense, and a version of this flowchart could be useful for introducing the modeling approach. (I'd leave off 1b for those purposes.) You're right, given the audience and the scope of your presentation, keeping the technical details to a minimum is smart. You might even consider leaving out eqns and R pseudo-code entirely, going directly from a verbal description of each model to a graphical presentation of its results (e.g., for 1, fitted regression plots and interval or joyplots of predicted additional rings). Is this a 15-min talk?

Where it gets a bit tricky is 4 & 5. As I alluded to above and elsewhere, there are several ways to summarize or graphically present the final output of the ring-correction analyses, and several ways to use it as input to the state-space model -- and the former doesn't necessarily need to match the latter. In fact, it hit me last night that perhaps the best way to summarize this input for the time-uncertain state-space model is *not* as a prior on total tree age, but as a prior on total ageing error, excluding the known rings. This would map directly onto the vignette (where the ageing error is represented by a simple geometric distribution for demo purposes), closing the gap between the toy model and our eventual real model that I've been fretting about. However, it might be more enlightening to graphically present the prior on tree age and/or annual recruitment, as in the current `total_age_priors.R`. 

The question, then, is how to phrase those last steps in the flowchart. Maybe there's a 6: Use informative ageing-error priors as input to state-space model. (IMO the state-space model should be in there, regardless.)

> I've already entered "stage fright" mode and will only become increasingly anxious and less able to think clearly as April 9th approaches. 

Haha, are you me?! I thought I was anomalous in experiencing this preemptive fight-or-flight brain-freeze.

> [I hear Bob Kaplan saying, "You have to play the game", and my response then as it is now is, "Actually, I don't."]

And I think of this often as well. Or as I remember him saying, "Stress is good." (Actually, [no it's not](https://www.bettermovement.org/blog/2014/a-systems-perspective-on-chronic-pain).)

**Comment by kzaret** [comment]
> You might even consider leaving out eqns and R pseudo-code entirely, going directly from a textual description of each model to a graphical presentation of its results (e.g., for 1, fitted regression plots and interval or joyplots of predicted additional rings). Is this a 15-min talk?

I have a 15-min. time slot, but the session organizers sent an email encouraging us to only talk for 10 mins. and leave 5 for questions/discussion.  I'm wrestling with whether to let go of talking about the overall research program so that I can give more background information that helps people understand conditions on the ground in SW Patagonia (including what I mean by 'forest-peatland ecotone').

I agree that leaving out equations and pseudocode is the way to go for this audience (and presenter).

> The question, then, is how to phrase those last steps in the flowchart. Maybe there's a 6: Use informative ageing-error priors as input to state-space model. (IMO the state-space model should be in there, regardless.)

I'm down with including a sixth step on a the flow chart; it's seems like it may be tricky to make clear how they differ from one another and/or to make them as tangible to the audience as steps 1 - 4 which can be illustrated with photos demonstrating sources of uncertainty or with the interval and/or joyplots.

> Haha, are you me?! I thought I was anomalous in experiencing this preemptive fight-or-flight brain-freeze.

I've sat down to work on this so many times over the past couple weeks and I just keep spinning my wheels, feeling so uncertain/indecisive about everything from the photo of the title slide (finally chosen) to whether to dive directly into the project location (orienting folks to SW Patagonia) or to take a larger view (e.g., point out that similar forest-peatland ecotones are found in SW Patagonia, Tasmania, New Zealand, or point out that though lots of climate-fire-veg work has been done using dendro methods, much of this has not occurred in both edaphicaly and climatically wet places).  I'm driving myself crazy.

I think the story I want to tell is something like this:  1) In SW Patagonia, where state policies encouraged the use of fire to clear land for settlement, there's been a dramatic shift in vegetation from cypress tree to peat moss dominance at numerous low-lying, poorly drained sites such that entire valley bottoms may be quite different today than they had been historically; 2) Past studies have found little evidence of forest regeneration following the 1970s, but given that Pilgerodendron uviferum is incredibly rot-resistant, we see stands of even-aged snags that would seem to represent historic post-fire cohorts; 3) Have thresholds in fire frequency and/or temperature and/or precipitation been passed such that P. uviferum forests are no longer able to recover from fire?; 4) By examining the timing of tree establishment relative to fire occurrence and large-scale climate modes that control local climate conditions, we can reveal whether anthropogenically-altered fire regimes, climate or their interactive effects are driving post-fire changes in structure of the forest-peatland ecotone of SW Patagonia; 5) Given challenges arising from the edaphically wet conditions at our study sites, we are building models of tree ring counts and tree age that account for various forms of measurement uncertainty with the goal of being able to work with annual-resolution data (since we'd like to be able examine whether there are spikes in tree establishment directly following fire episodes). . . .

**Comment by ebuhle** [comment]
> I'm down with including a sixth step on a the flow chart; it's seems like it may be tricky to make clear how they differ from one another and/or to make them as tangible to the audience as steps 1 - 4 which can be illustrated with photos demonstrating sources of uncertainty or with the interval and/or joyplots.

What do you think about including some of the plots from the vignette illustrating the time-uncertain state-space model fitted to simulated data?

> I think the story I want to tell is something like this:

As a more or less naive observer, this outline / intro makes sense to me. One question, for my own understanding:

> 2) Past studies have found little evidence of forest regeneration following the 1970s, but given that Pilgerodendron uviferum is incredibly rot-resistant, we see stands of even-aged snags that would seem to represent historic post-fire cohorts

So they're even-aged because they established following fire, and they're dead because of a subsequent fire? Another question, born of idle curiosity:

> similar forest-peatland ecotones are found in SW Patagonia, Tasmania, New Zealand

Where in NZ? (Also, it seems like both of your examples of broader context could fit naturally into the story you outlined without taking up much extra time.)

> I'll be presenting during the Dendro I session that runs from 8 am to 9:15 am [uufff] on 4/9.

Brutal, but I just learned Rebekah Jones's [keynote](https://www2.aag.org/aagannualmeeting/special_events) is the hour before this, so you'll be up anyway, right? :wink: I also learned Saint Naomi is making an appearance Thursday afternoon!

**Comment by kzaret** [comment]
> What do you think about including some of the plots from the vignette illustrating the time-uncertain state-space model fitted to simulated data?

Sound good, if you tell me what you want me to say about them ;-)

>So they're even-aged because they established following fire, and they're dead because of a subsequent fire?

Yes. It seems that P. uviferum has multiple regeneration strategies:  the tree can (if the conditions are right [whatever that means]) establish after intense or catastrophic disturbance, forming even-aged cohorts, or it can establish continuously under a fairly closed canopy with individuals undergoing growth release as canopy gaps are created by the death of older/larger trees; oh yeah, and new stems can form from adventitious roots.

> Where in NZ?

I'll email you a .kmz of sites mentioned by Johnson (2005) in "Fire in wetlands and scrub vegetation:  studies in Southland, Otago, and Wetland" published by the Department of Conservation.  Basically, on the northern/wetter side of the Southern Alps.  It also sounds like there are sites in Northland where Agathis australis stands were burned to harvest an amber-like resin called "kauri gum" (kauri is the common name of A. australis).  Burning and/or logging in mesic to wet A. australia stands seems to have led to shifts from forest to pakihi, which is type of wet heath that's sometimes on peat soils or sometimes peat-forming:  a cool parallel to the P. uviferum forest to peat moss dominated site story.

>(Also, it seems like both of your examples of broader context could fit naturally into the story you outlined without taking up much extra time.)

Would you insert them in the introduction or try to circle back around to them at the end?  (I think I may find the latter hard -- to pull back from the state-space model that far.)

**Comment by kzaret** [comment]
Saint Naomi's talk has been on my calendar for a while ;-)

**Comment by kzaret** [comment]
For a 15 min. talk (with only 10 minutes of speaking time) would you/do you bother adding a slide outlining the talk?

**Comment by ebuhle** [comment]
> Sound good, if you tell me what you want me to say about them ;-)

I was thinking of the 4-panel figure at the end that illustrates the state-space concept (state process and observation errors); the further corruption of observed time indices by ageing error "jitter"; estimation of the unknown latent process by fitting the state-space model to the observations; the bias in those estimates introduced by the ageing error; and the ability of the time-uncertain model to do a decent job of accounting for the ageing error and recovering the process.

Obviously you're not going to explain the mathematical flute music, but I'd envisioned this as a closing teaser or preview of where the analysis is going and an intuitive hand-waving introduction of (possibly unfamiliar) concepts to the audience. You could even play the "ask / blame my quantitative coauthor" line for a laugh.

I'd be happy to suggest talking points or make notes on this or other slides. For this one, you could start with the short and breezy paragraph in the vignette that explains the figure in question. I'll go ahead and write it out to a file, anyway (currently it only exists in the rendered document).

> For a 15 min. talk (with only 10 minutes of speaking time) would you/do you bother adding a slide outlining the talk?

Depends on the audience and the complexity of the content. For this one, especially since you have that nice flowchart, I'd probably just talk through the road map over the title slide and then jump into the place-based narrative (which I'm guessing will be enough of a hook for the audience at a dendro session). Possibly I'd have a different take if I saw your slides.

**Comment by kzaret** [comment]
> I was thinking of the 4-panel figure at the end that illustrates the state-space concept (state process and observation errors); the further corruption of observed time indices by ageing error "jitter"; estimation of the unknown latent process by fitting the state-space model to the observations; the bias in those estimates introduced by the ageing error; and the ability of the time-uncertain model to do a decent job of accounting for the ageing error and recovering the process.

> I'd be happy to suggest talking points or make notes on this or other slides. For this one, you could start with the short and breezy paragraph in the vignette that explains the figure in question. I went ahead and wrote it out to a file, anyway (previously it only existed in the rendered document).

Thank you!

Taking a look at this -- the 4 panel figure -- I think I'm going to struggle explaining this and people are going to have trouble understanding it unless it's broken down for them/us even more than in the description in the vignette.  I wonder whether the following plot from the vignette would help warm folks up:  ![image](https://user-images.githubusercontent.com/73861013/113919325-531a7c80-9798-11eb-98a1-b2cccbe505ed.png)

Could you remind me of what lamda t represents?

**Comment by ebuhle** [comment]
Well, first of all it's `gamma[i,t]`. :wink: And it's the observation error likelihood -- specifically, the probability that the observed time (e.g. establishment year) for the *i*th event (e.g. tree), `t[i]`, is equal to `t` (the *x*-axis), given that the true time is `tau[i]`. The jitter, if you like. In the toy model for simulation testing, I made this likelihood a simple geometric aka discrete exponential distribution with a known scale parameter, i.e. a point-mass prior on the true observation model that generated the data from the underlying discrete latent states `tau`. This corresponds directly to the posterior distribution of "total additional rings" in our analysis. The probability of *k* unobserved rings is the probability that the observed age (known, minimum ring count) is *k* years younger than (to the right of) the true age.

I didn't reckon you'd want this one, but I've now saved it to a [file](https://github.com/kzaret/RQ1v2_PIUVestab/blob/main/analysis/results/plot_geom_obs-1.png) as well.
